<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ATMECSINDT-080" failures="0" tests="3" name="com.atmecs.MercuryTripa.atmecs.testscripts.scenario1.AboutUs" time="23.147" errors="1" timestamp="17 Oct 2019 07:05:18 GMT" skipped="0">
  <testcase classname="com.atmecs.MercuryTripa.atmecs.testscripts.scenario1.AboutUs" name="setup" time="20.715"/>
  <testcase classname="com.atmecs.MercuryTripa.atmecs.testscripts.scenario1.AboutUs" name="ClickOnAboutUs" time="2.428"/>
  <testcase classname="com.atmecs.MercuryTripa.atmecs.testscripts.scenario1.AboutUs" name="footervalidationAboutUs" time="0.004">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.atmecs.MercuryTripa.org.atmecs.helpers.ValidationOfFooter.footerOfAtmecs(ValidationOfFooter.java:31)
at com.atmecs.MercuryTripa.atmecs.testscripts.scenario1.AboutUs.footervalidationAboutUs(AboutUs.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- footervalidationAboutUs -->
</testsuite> <!-- com.atmecs.MercuryTripa.atmecs.testscripts.scenario1.AboutUs -->
